### ISSUE TITLE
Allow us to specify which Ollama model to use

### ISSUE THREAD

**Issue by aviaryan** [jacoblee93/fully-local-pdf-chatbot#27]
I don't think there is a hard requirement to only use Mistral.
Can we have a feature to use any Ollama model?
I would like to use llama3 or phi3 for example.

<img width="360" alt="Screenshot 2024-08-01 at 7 56 23 PM" src="https://github.com/user-attachments/assets/de14af04-2cb6-49c4-84be-f83d3c420127">

**Comment by jacoblee93** [comment]
Yes would be nice! As a short term workaround you can modify the source code.

**Comment by aviaryan** [comment]
Yes, I did that. Thanks!

**Comment by iqs-systems** [comment]
Can you show me where is I can change source code?

**Comment by jacoblee93** [comment]
Here:

https://github.com/jacoblee93/fully-local-pdf-chatbot/blob/main/components/ChatWindow.tsx#L116
